kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/QuestionListPreview.vue

```diff
@@ -215,4 +395,52 @@
     opacity: 0;
   }
 
+  .accordion-header-label {
+    display: block;
+    width: calc(100% - 1em);
+    height: 100%;
+    padding: 1em;
+
+    // Removes underline from section headings
+    /deep/.link-text {
+      text-decoration: none;
+    }
+  }
+
+  .chevron-icon {
+    position: absolute;
+    top: 50%;
+    right: 0.5em;
+    vertical-align: middle;
+    transform: translateY(-50%);
+  }
+
+  .accordion-header {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+    font-size: 1rem;
+    line-height: 1.5;
+    text-align: left;
+    cursor: pointer;
+    user-select: none;
+    transition: background-color 0.3s ease;
+  }
+
+  .question-button {
+    width: 100%;
+    height: 100%;
+    padding: 0.5em;
+
+    &:hover {
+      background-color: white;
+    }
+
+    &.selected {
+      background-color: white;
+    }
+  }
```

Comment:
A bit of a stop-gap, just wanted to get this PR made, will include notes on this in a tech-debt follow-up